templates/addons/windows/kube-proxy-windows.yaml

```diff
@@ -57,7 +62,9 @@ spec:
           valueFrom:
             fieldRef:
               fieldPath: status.podIP
-        image: sigwindowstools/kube-proxy:${KUBERNETES_VERSION}-nanoserver
+        - name: USE_CI_VERSION
+          value: "${USE_CI_VERSION:false}"
+        image: sigwindowstools/kube-proxy:${KUBERNETES_VERSION/+/_}-nanoserver
```

Comment:
any plans to migrate away from dockerhub?

Reply:
yes, once we move to HostProcess containers we will build these images just like the Linux Kube-proxy image.